### PR TITLE
F 2301 add monitoring to feathers services

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
If you want to send monitoring data to sentry you should add `"enableMonitoring":true` to config

### CODE REVIEW

**Comment by mohammadranjbarz** [comment]
related to Giveth/giveth-dapp#2301

File: config/default.json [hunk 1]
```diff
@@ -176,5 +176,6 @@
   "elasticSearchUsername": "elastic",
   "elasticSearchPassword": "changeme",
   "enableAuditLog": false,
+  "enableMonitoring": true,
```
Comment (aminlatifi) [inline config/default.json:179]: please rename it to some variable holding sentry name

**Comment by aminlatifi** [comment]
@mohammadranjbarz I changed the variable name and its initial value

**Comment by mohammadranjbarz** [comment]
> @mohammadranjbarz I changed the variable name and its initial value

Thanks man

**Pull request merged**